pkg/network/infraconfigurators/masquerade.go

```diff
@@ -382,6 +382,10 @@ func (b *MasqueradePodNetworkConfigurator) createNatRulesUsingNftables(proto ipt
 			if err != nil {
 				return err
 			}
+			err = b.forwardPortUsingNftables(proto, fmt.Sprintf("%d", istio.SshPort))
```

Comment:
> Instead of hard coding here port 22, I would add a method similar to istio.ReservedPorts() maybe
> istio.NonProxiedPorts.

It is good you moved the ssh port to the istio package but I would take it even one step forward and would add a method `istio.NonProxiedPorts`. I think it will make the code more readable instead of trying to understand was is so special with the ssh port.